astropy/io/fits/header.py

```diff
@@ -696,16 +703,18 @@ def fromtextfile(cls, fileobj, endcard=False):
 
         return cls.fromfile(fileobj, sep='\n', endcard=endcard, padding=False)
 
-    def totextfile(self, fileobj, endcard=False, clobber=False):
-        """
+    @deprecated_renamed_argument('clobber', 'overwrite', '1.3')
+    def totextfile(self, fileobj, endcard=False, overwrite=False):
+        """Write the header as text to a file or a file-like object
+
```

Comment:
It's an alias for another function so it makes sense not to have the parameter list copied here.

👍 for putting it in the overall docstring.